android/sdk/src/main/cpp/app/organicmaps/sdk/bookmarks/data/BookmarkManager.cpp

```diff
@@ -824,14 +834,12 @@ Java_app_organicmaps_sdk_bookmarks_data_BookmarkManager_nativeRemoveElevationCur
 }
 
 JNIEXPORT void JNICALL Java_app_organicmaps_sdk_bookmarks_data_BookmarkManager_nativeSetElevationActivePoint(
-    JNIEnv *, jclass, jlong trackId, jdouble distanceInMeters)
+    JNIEnv *, jclass, jlong trackId, jdouble distanceInMeters, jdouble latitude, jdouble longitude)
 {
   auto & bm = frm()->GetBookmarkManager();
-  bm.SetElevationActivePoint(
-      static_cast<kml::TrackId>(trackId),
-      {0,
-       0},  // todo(KK): replace with coordinates from the elevation profile point to show selection mark on the track
-      static_cast<double>(distanceInMeters));
+  bm.SetElevationActivePoint(static_cast<kml::TrackId>(trackId),
+                             m2::PointD(static_cast<double>(latitude), static_cast<double>(longitude)),
+                             static_cast<double>(distanceInMeters));
```

Comment:
Is static cast really needed?
```suggestion
                             {latitude, longitude}, distanceInMeters);
```